nautobot/core/testing/views.py

```diff
@@ -1560,3 +1560,24 @@ def test_bulk_rename(self):
                     response = self.client.post(self._get_url("bulk_rename"), data, follow=True)
                     expected_message = f"No valid {verbose_name_plural} were selected."
                     self.assertBodyContains(response, expected_message)
+
+        @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
+        def test_modular_component_create_form_fields(self):
```

Comment:
From the CI failure, looks like this needs to account for the difference between `ComponentCreateForm` (e.g. DeviceBayCreateForm) and `ModularComponentCreateForm` views.

Reply:
I added some logic to account for this. The ModuleDeviceBay add view also has `Required device` rather than just `Device` so I tried to handle that as well.